src/scripts/services/user-service.ts

```diff
@@ -109,34 +110,54 @@ export class OktaUserService {
   };
 
   // eslint-disable-next-line functional/functional-parameters
-  readonly listUsers = (): TE.TaskEither<string, readonly User[]> => {
+  readonly listUsers = (
+    // eslint-disable-next-line @typescript-eslint/prefer-readonly-parameter-types
+    maybeGroup: O.Option<Group> = O.none
+  ): TE.TaskEither<string, readonly User[]> => {
     // We need to populate users with all of the client data so it can be
     // returned. Okta's listUsers() function returns a custom collection that
     // does not allow for any form of mapping, so array mutation is needed.
 
-    return TE.tryCatch(
-      // eslint-disable-next-line functional/functional-parameters
-      () => {
-        // eslint-disable-next-line functional/prefer-readonly-type
-        const users: User[] = [];
+    // eslint-disable-next-line functional/prefer-readonly-type
+    const users: User[] = [];
```

Comment:
Move this down to line 146. Since this is a gross thing, we want to scope it as tightly as possible.